Externals.cfg

```diff
@@ -30,7 +30,7 @@ protocol = git
 repo_url = https://github.com/NOAA-EMC/UPP
 # Specify either a branch name or a hash but not both.
 #branch = develop
-hash = 394917e
+hash = fbd41a5
 local_path = src/UPP
 required = True
```

Comment:
@JeffBeck-NOAA, this hash has been tested with PR #266 on other machines. Once this PR is approved, I'll change this hash to the original in this PR. This hash is just for testing.

Reply:
Tested on wcoss2. Build was successful.

Reply:
What is the status of these hashes on Cheyenne?  @mkavulich, @mark-a-potts, do we need to update hpc-stack or otherwise install newer libraries for this PR and for PR #266?